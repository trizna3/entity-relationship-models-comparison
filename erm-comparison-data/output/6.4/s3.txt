trips-trips
users-users
points-coordinates
pictures-EMPTY
REBIND_1NN1_TO_MN:S-rating
