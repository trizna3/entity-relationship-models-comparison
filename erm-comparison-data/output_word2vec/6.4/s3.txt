trips-trips
points-edges;coordinates
users-users
pictures-images
CHANGE_CARDINALITY:S-edges-coordinates
CONTRACT_11_ASSOCIATION:S-edges-coordinates
