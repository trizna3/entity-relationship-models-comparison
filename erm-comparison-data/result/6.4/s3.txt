Mapping 3/4 pairs, 1 additional pairs.
Missing pairs: 
 pictures-images
Additional pairs: 
 pictures-empty

Mapping  0/1 transformations, 1 additional transformations.
Missing transformations: 
 rebind_1nn1_to_mn:s-ratings
Additional transformations: 
 rebind_1nn1_to_mn:s-rating
