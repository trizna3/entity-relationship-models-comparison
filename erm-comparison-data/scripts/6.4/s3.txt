entity set users {
	"name"
	"surname"
	"username"
	"password"
};

entity set trips {
	"text"
	"rating"
};

entity set rating {
	"stars"
};

entity set edges {
	"sequence number"
};

entity set coordinates {
	"x"
	"y"
};

entity set images {
	"link"
	"description"
};

association "shares" between 
{
	users ONE 
	trips MANY
}{};

association between 
{
	users ONE 
	rating MANY
}{};

association between 
{
	rating MANY 
	trips ONE
}{};

association between 
{
	trips ONE 
	edges MANY
}{};

association between 
{
	trips ONE 
	images MANY
}{};

association "start point" between 
{
	edges MANY 
	coordinates ONE
}{};

association "end point" between 
{
	edges MANY 
	coordinates ONE
}{};

association between 
{
	coordinates ONE 
	images MANY
}{};