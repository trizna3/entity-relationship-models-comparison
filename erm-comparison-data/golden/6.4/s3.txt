users-users
trips-trips
pictures-images
points-coordinates
REBIND_1NN1_TO_MN:S-ratings